### PR TITLE
Update device-services.md

### PR DESCRIPTION
**Pull request by sh783**
Fixed typo

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/managed-desktop/service-description/device-services.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Microsoft will provide these services for Microsoft Managed Desktop devices. For
  --- | ---
 Support	| Support agents will answer questions directly related to device functionality and diagnose device issues.
 Inventory |	All devices are tracked in the Microsoft Managed Desktop Admin portal for inventory and status tracking.
-Firmware / driver updates |	By default, Microsoft Managed Desktop devices receive firmware and driver updates from Windows Update. Not all hardware partners deploy their updates via Windows Update. Updates not published as Automatic require an exception and must deployed by the customer.
+Firmware / driver updates |	By default, Microsoft Managed Desktop devices receive firmware and driver updates from Windows Update. Not all hardware partners deploy their updates via Windows Update. Updates not published as Automatic require an exception and must be deployed by the customer.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/device-services.md:24]:
```suggestion
Firmware/Driver updates |	By default, Microsoft Managed Desktop devices receive firmware and driver updates from Windows Update. Not all hardware partners deploy their updates via Windows Update. Updates not published as Automatic require an exception and must be deployed by the customer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit in accordance with MS Style Guide.

**Comment by jaimeo** [comment]
@sh783 - @ me in these things so I get notified and then you won't have to message me.

**Comment by sh783** [comment]
Thanks for the reminder.  What’s your @ in github?

**Comment by jaimeo** [comment]
@sh783 - look at my comment just above. Or this one :^)

**Comment by sh783** [comment]
Just so you know, this is what I was seeing:

[A screenshot of a social media post  Description automatically generated]

I opened it in GitHub to see what you were highlighting.  Will tag jaimeo going forward.   😊

**Pull request merged**